scripts/add_ts_data_bidmatrixinfo.py

```diff
@@ -0,0 +1,80 @@
+###this script adds time series data to the bid matrix information objects in the powerops client
+
+from cognite.client.data_classes import TimeSeries
+from cognite.powerops import PowerOpsClient
+import random
+import datetime
+
+
+power = PowerOpsClient.from_settings()
+cdf = power.cdf
+
+start_time = int((datetime.datetime.now() - datetime.timedelta(days=365)).timestamp() * 1000)
+end_time = int(datetime.datetime.now().timestamp() * 1000)
+
+exid_hourly = "wim_ts_hourly"
+exid_empty = "wim_ts_empty"
+
+# Checking if the time series already exists
+existing_time_series = cdf.time_series.retrieve_multiple(external_ids=[exid_hourly, exid_empty])
+
+existing_time_series_ids = []
+for ts in existing_time_series:
+    existing_time_series_ids.append(ts.external_id)
+
+
+# Adding the time series to a list, if they don't exist
+time_series_to_create = []
+
+if exid_hourly not in existing_time_series_ids:
+    time_series_to_create.append(TimeSeries(name="wim_ts_hourly", external_id=exid_hourly))
+    data_points = [
+        {"timestamp": start_time + i * 3600 * 1000, "value": random.random()}
+        for i in range(int((end_time - start_time) / (3600 * 1000)))
+    ]
+    cdf.time_series.data.insert(datapoints=data_points, external_id=exid_hourly)
+
+if exid_empty not in existing_time_series_ids:
+    time_series_to_create.append(TimeSeries(external_id=exid_empty))
+
+# Creating the time series if the list has any elements
+if time_series_to_create:
+    cdf.time_series.create(time_series_to_create)
+
+tsList = [exid_hourly, exid_empty]
+
+
+def get_random_timeseries():
```

Comment:
**note:** if a function relies on a variable that's defined outside of the function (ie `tsList`) it's best to define that as an input argument so that you can explicitly pass in the variable you want to use instead of implicitly relying on it to be defined somewhere outside of the function